tools/sgcloudrun/config.go

```diff
@@ -219,3 +300,73 @@ func accessSecretVersion(ctx context.Context, accessToken, project, secret, vers
 	}
 	return string(decodedData), nil
 }
+
+func applicationDefaultCredential() (string, error) {
```

Comment:
applicationDefaultCredentials* (they use plural in Google's APIs)

Reply:
Fixed